docs/lang/articles/deployment/tutorial.md

```diff
@@ -4,185 +4,227 @@ sidebar_position: 1
 
 # Tutorial: Run Taichi programs in C++ applications
 
-Taichi makes it easy to write high-performance programs with efficient parallelism, but in many applications we cannot simply deploy the Python scripts. Taichi offers a runtime library (TiRT) with a C interface so that your Taichi kernels can be launched in any native application. In this tutorial, we'll walk through the steps to deploy a Taichi program in a C++ application.
+Taichi makes it easy to write high-performance programs with efficient parallelism, but in many applications we cannot simply deploy the Python scripts. Taichi offers a runtime library (TiRT) with a C interface as well as its C++ wrapper, so your Taichi kernels can be launched in any native application. In this tutorial, we'll walk through the steps to deploy a Taichi program in a C++ application.
```

Comment:
do we have any commitment on the compatibility/stability of C++ wrapper? :P cc: @PENGUINLIONG

Reply:
It should tries its best to ensure it compiles across versions. But it's not officially 'stablized' atm. (Baiscally that I don't have enough time to write apps with it and experience it thoroughly. Maybe some time later.)